test/test_generator.py

```diff
@@ -18,12 +26,12 @@
 )
 from stone.generator import CodeGenerator
 
-class Tester(CodeGenerator):
```

Comment:
Because these started with `Test`, some tools assumed they are derived from `unittest.TestCase`. Hence, the rename.